chrome/0004-intermediete-mappings-cases.patch

```diff
@@ -0,0 +1,1421 @@
+commit ad588a7e074c1fa371a1a6bb2ab3b1f243943f3f
+Author: Agata Belkius <abelkius@gmail.com>
+Date:   Wed May 29 17:51:23 2024 +0100
+
+    add test cases for intermediate mappings, move all tests to one it block
+
+diff --git a/front_end/core/sdk/SourceMapSpec.test.ts b/front_end/core/sdk/SourceMapSpec.test.ts
+index 2d6818f785..22e6246674 100644
+--- a/front_end/core/sdk/SourceMapSpec.test.ts
++++ b/front_end/core/sdk/SourceMapSpec.test.ts
```

Comment:
most changes are in this file, the rest is basically updating the test resources which come from this repo.